cli/read.go

```diff
@@ -162,7 +162,7 @@ func printPayload(index int, p handlers.Installer) {
 	fmt.Printf("    Provides:")
 	if err != nil {
 		fmt.Printf(" Invalid provides section: %s\n", err.Error())
-	} else if provides == nil || len(provides) == 0 {
+	} else if len(provides) == 0 {
```

Comment:
Q: Why is this needed? :thinking:

Reply:
Then please keep it in a seperate commit @0lmi  :)

Reply:
👍